be/src/olap/compaction.cpp

```diff
@@ -134,7 +134,7 @@ Status Compaction::do_compaction(int64_t permits) {
                          << ", before=" << checksum_before << ", checksum_after=" << checksum_after;
         }
     }
-    _load_segment_to_cache();
+    RETURN_IF_ERROR(_load_segment_to_cache());
```

Comment:
It's not necessary to return error here?
`_load_segment_to_cache` should be a best effort optimization, when it fail, should just impact some performance, but not make the compaction job fail.